build.gradle

```diff
@@ -1,6 +1,14 @@
 group 'multi-int-store'
 version '0.1.0-SNAPSHOT'
 
-repositories {
-    mavenCentral()
-}
\ No newline at end of file
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'org.owasp:dependency-check-gradle:4.0.1'
+    }
+}
+
+apply plugin: 'org.owasp.dependencycheck'
```

Comment:
❗️Add documentation in the README.md about how to run OWASP

Reply:
I'll append my docs ( `gradle dependencyCheckAnalyze --info`) once @jhunzik 's PR (https://github.com/connexta/multi-int-store/pull/3)  gets in. I think he's set up the Help/README nicely.

Reply:
That README was generated, but we can steal the format 😄

Reply:
There is already a README in the repo. You don't need to wait for Josh.

Reply:
❓ Does this get ran automatically with `./gradlew build`?

Reply:
No. You have to so the `gradle dependencyCheckAnalyze --info` command to run the OWASP check. We don't want it to run during `./gradlew build` so that it doesn't affect developers, only CI builds.